pkg/util/port_split_test.go

```diff
@@ -32,13 +32,25 @@ func TestSplitPort(t *testing.T) {
 			port:  "asdf",
 			valid: true,
 		}, {
+			in:     "aoeu:asdf:http",
+			name:   "aoeu",
+			port:   "asdf",
+			scheme: "http",
+			valid:  true,
+		}, {
+			in:     "aoeu:asdf:https",
+			name:   "aoeu",
+			port:   "asdf",
+			scheme: "https",
+			valid:  true,
+		}, {
 			in:    "aoeu:",
 			name:  "aoeu",
 			valid: true,
 		}, {
 			in:   ":asdf",
 			name: "",
-			port: "asdf",
+			port: "",
```

Comment:
Although not your change, you should add valid:false to the invalid test cases to make it obvious that's what's expected.